web/src/js/ducks/view.js

```diff
@@ -0,0 +1,31 @@
+import { SELECT_FLOW } from './flows'
+const ACTIVE_MENU = 'ACTIVE_MENU'
+
+
+const defaultState = {
+    active_menu: 'Start'
+}
+export default function reducer(state = defaultState, action) {
+    switch (action.type) {
+        case ACTIVE_MENU:
+            return {
+                ...state,
+                active_menu: action.active_menu
+            }
+        case SELECT_FLOW:
+            return{
+                ...state,
+                active_menu: action.flowId ? 'Flow' : 'Start'
```

Comment:
yes i understand that problem, i think i just use the privious state and go for 2.